docs/remote/containers.md

```diff
@@ -170,7 +170,7 @@ For example, follow these steps to open one of the "try" repositories in a Repos
 
     ![PR status in status bar](images/containers/checkout-pr-status.png)
 
-5. After the build completes, VS Code will automatically connect to the container. You can now work with the repository source code in this isolated environment as you would if you had cloned the code locally.
+5. After the build completes, VS Code will automatically connect to the container. You can now work with the repository source code in this independent environment as you would if you had cloned the code locally.
```

Comment:
Why do we want to update to "independent" and "separate" rather than isolated? In my explanations, I typically refer to them as isolated and reproduceable environments.

Reply:
Does something like separate or independent feel more applicable to Codespaces?

Reply:
Isolated implies it is completely separate from the host operating system.  On Windows and macOS when you're using Docker Desktop this is true.  However, on Linux, it's not given docker can run as root. So I don't want to oversell. They're definatley isolated in concept, but this also tends to be a key word in security conversations that means more.

Reply:
Also, FYI - Christof and I are talking about rootless Docker [here](https://github.com/microsoft/vscode-remote-release/issues/4646). I'm seeing some limitations with it for bind mounting sadly.

Reply:
Makes sense - thanks for the insight!